deployments/helm-chart/templates/controller-daemonset.yaml

```diff
@@ -145,7 +145,11 @@ spec:
           - -app-protect-dos-max-workers={{ .Values.controller.appprotectdos.maxDaemons }}
           - -app-protect-dos-memory={{ .Values.controller.appprotectdos.memory }}
   {{ end }}
+{{- if .Values.controller.customConfigMap }}
+          - -nginx-configmaps=$(POD_NAMESPACE)/{{ .Values.controller.customConfigMap }}
+{{- else }}
           - -nginx-configmaps=$(POD_NAMESPACE)/{{ include "nginx-ingress.configName" . }}
+{{- end }}
```

Comment:
I just realized my internet died while I was posting the second comment and it never went through 😭 

I think we can have a helper function in `deployments/helm-chart/templates/_helpers.tpl` to use `customConfigMap` if set or the default one otherwise. 

We should also clarify this in the docs, that if the custom one is set, the other is ignored.

Reply:
Updated. Let me know if I should change the verbiage.